docs/UserGuide.md

```diff
@@ -120,325 +127,416 @@ Displays the output of `View` command, which expands each ingredient or recipe c
 
 Clear all the ingredients from the Inventory. Add `expired` keyword to only clear expired ingredients.
 
 **Format:**<br />
 `clear ingredient [expired]`
 
 **Example(s):**<br />
-1. `clear ingredient`
+1. Clear all ingredients:<br />
+Command: `clear ingredient`
 <br />Expected Output:<br />
 ![clearCommand.png](images/ingredientCommands/clearCommand.png)
-2. `clear ingredient expired`
-<br />Expected output:<br />
-![clearCommand2.png](images/ingredientCommands/clearCommand2.png)
+2. Clear expired ingredients:<br /> 
+Command: `clear ingredient expired`
+<br />Before:<br />
+![clearCommand2.png](images/ingredientCommands/clearExpired1.png)
+<br />After:<br />
+![clearCommand3.png](images/ingredientCommands/clearExpired2.png)   
+
+[\[Back to Table of Contents\]](#table-of-contents)
 
-### 4.2.5 Find Ingredients
+#### 4.4.5 Find Ingredient
 
-- Search for an ingredient from the Inventory based on a user-inputted keyword(s) that match the name of an ingredient(s).
-- After [Find Ingredient](#425-find-ingredients) command, to see the full list of ingredients again, please use
-[List Ingredient](#426-list-ingredients) command.
+- Search for ingredient(s) from the Inventory based on user-inputted keyword(s) that match the **name** of ingredient(s).
+- After a [Find Ingredient](#445-find-ingredient) command, to see the full list of ingredients again, please use a
+[List Ingredient](#446-list-ingredient) command.
 
 **Format:**<br />
 `find ingredient <keyword>...`
 
 **Example(s):**<br />
-1. `find ingredient cream`
-   <br />Expected Output:<br />
-   ![findIngredient1.png](images/ingredientCommands/findIngredient1.png)
+1. Command: `find ingredient banana cream`
+<br />Expected Output:<br />
+![findIngredient1.png](images/ingredientCommands/findIngredient.png)
 
 **Additional Information:**<br />
-- Requirements for a keyword:
-    1. Keyword is case-insensitive.<br />
+- Keyword(s) are separated by spaces.
+  - e.g. "corn chicken beef" will be regarded as "corn", "chicken" and "beef". 
+Results will show matches for any of the 3 keywords.
+
+- Match Criteria:
+    1. Matching of keyword(s) is case-insensitive.<br />
        e.g. Finding with keyword: "corn" will match with "COrN"
-    2. Any keyword must be contained in the name of the recipe.<br />
-       e.g. Finding with keyword: "corn" will match with "CoRN flour", "coRn FlakeS", "popcorn" but not "`" <br />
-       e.g. Finding with keywords: "corn Chicken beef" will match "beef Chicken", "beef corn", etc. but not "beefcorn"
+    2. Ingredient(s) will be matched as long as a full keyword is in its name.<br />
+       e.g. Finding with keyword: "corn" will match with "cornflour" and "corn flakes" but not "apple core". <br />
 
-### 4.2.6 List Ingredients
+[\[Back to Table of Contents\]](#table-of-contents)
 
-List all the ingredients again after `find ingredient` Operation.
+#### 4.4.6 List Ingredient
+
+Switch to Ingredient [tab](#31-tabs) and list all ingredients.
```

Comment:
Mention sorting of ingredients smwhere? Hmm actually not sure where to put the sorting info cuz its across the commands right.

Reply:
It's under 4.4

Reply:
Ah ok